perl/dist.ini

```diff
@@ -0,0 +1,48 @@
+name      = Gherkin
+abstract  = A parser and compiler for the Gherkin language
+main_module = lib/Gherkin.pm
+author    = ['Peter Sergeant <pete@clueball.com>', 'Gaspar Nagy']
+license   = MIT
+is_trial  = 0
+perl      = 5.008
+copyright_holder = Peter Sergeant, Gaspar Nagy
+
+; Worth seeing the note about this in Gherkin.pm. You are aiming for the Gherkin
+; version with the minor revision %2d and a two-digit CPAN version added. For
+; example:
+;
+;   3.1.2 -> 3.1.0201, 3.1.0202, 3.1.0203 ... etc
+;   3.2.0 -> 3.2.0001, 3.2.0002 ... etc
+;
+; CPAN releases can't be changed, and this allows a nice simple way of uploading
```

Comment:
> CPAN is particularly unforgiving about allowing rereleases.

Almost all package repositories are, and I think that is a good thing.

> Do you really want to do a whole new central Gherkin release because the Perl version can't be installed on 50% of the machines due to some Perl weirdery?

Our release cycle is currently roughly every month. Between each release there are usually several changes across platforms, so I don't see this as a problem. If there are bugs in the perl implementation we'll fix them, but it probably won't affect the release cycle. People who can't wait will have to grab git master :-)